examples/albef/data/vqa_dataset.py

```diff
@@ -0,0 +1,118 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import json
+import os
+import re
+from typing import List, Tuple, Union
+
+from examples.albef.data.transforms import ALBEFTransform
+
+from PIL import Image
+from torch import Tensor
+from torch.utils.data import Dataset
+
+
+class VQADataset(Dataset):
+    """
+    Create the dataset for VQA task
```

Comment:
Add more detail here (especially around the difference between what gets returned in train and test sets)